guide/interactions/autocomplete.md

```diff
@@ -0,0 +1,110 @@
+# Autocomplete
+
+Slash command options provide the feature of autocomplete. In this section, we'll cover how to add autocomplete support to your command and how to respond to it.
+
+::: tip
+This page is a follow-up to the [interactions (slash commands) pages](/interactions/registering-slash-commands.md). Please carefully read those first so that you can understand the methods used in this section.
+:::
+
+::: warning
+Make sure that you're on discord.js version 13.3.0 or above before using autocomplete.
+:::
+
+## Preparing commands
```

Comment:
```suggestion
```
This feels off, what is being prepared? This entire paragraph seems redundant as the action of setting the property to true is explained below already.

Reply:
Should `Preparing commands` be replaced with `Enabling` or `Enabling autocomplete` or should this section just be put in under `Receiving`?

Reply:
merging those would probably be most senseful unless a a more appropriate header name was used